src/app/map/page.tsx

```diff
@@ -1,3 +1,21 @@
-export default function MapPage() {
-    return <div>This is map demo</div>
+"use client"
+
+import StaticMap from "@/components/StaticMap";
+
+import InteractiveMap from "@/components/InteractiveMap";
+
+// import dynamic from "next/dynamic";
+
+// const DynamicInteractiveMap = dynamic(() => import("@/components/InteractiveMap"), {
+//   ssr: false,
+// });
```

Comment:
Co to za komentarze

Reply:
Tu generalnie chodzi o to, że zakomentowany kod będzie na produkcji, bo po buildowaniu bez dynamic importa wywala błąd przy obecnym kodzie.

Reply:
Z tym ze to i tak nie rozwiazuje problemu tego unmountingu z tego co rozumiem

Reply:
Jesli bedziesz ostatecznie uzywac tego dynamic exportu to trzeba bedzie go troche zmodyfikowac jesli tam jest named export
```suggestion
// import dynamic from "next/dynamic";

// const DynamicInteractiveMap = dynamic(() => import("@/components/InteractiveMap").then(mod => mod.InteractiveMap, {
//   ssr: false,
// });
```

Reply:
Dobra, zmianę wprowadziłem do importów, ale z tym unmountingiem do jutra sie nie wyrobie z researchem, bo mam rano kolokwium i jeszcze sie musze poduczyć na nie. Spróbuje jeszcze po kolosie coś ogarnąć, ale może być ciężko